collectors/cronjob.go

```diff
@@ -120,8 +153,12 @@ func (jc *cronJobCollector) collectCronJob(ch chan<- prometheus.Metric, j v2batc
 		addGauge(descCronJobSpecStartingDeadlineSeconds, float64(*j.Spec.StartingDeadlineSeconds))
 	}
 
-	addGauge(descCronJobInfo, 1, j.Spec.Schedule, string(j.Spec.ConcurrencyPolicy))
+	// If the cron job is suspended, don't track delays
+	if j.Status.LastScheduleTime != nil && !*j.Spec.Suspend {
```

Comment:
What happens to the metric if a CronJob has never been successfully scheduled, for example if it's too big for any machine in the cluster? I think we want to alert on that also.